sockeye/data_io.py

```diff
@@ -451,6 +472,8 @@ def load(self,
                        for (source_len, _), num_samples in zip(self.buckets, num_samples_per_bucket)]
         data_target = [np.full((num_samples, target_len + 1, num_target_factors), self.pad_id, dtype=self.dtype)
                        for (_, target_len), num_samples in zip(self.buckets, num_samples_per_bucket)]
+        # metadata contains the length of prepended tokens
+        data_meta = [np.full((num_samples, 1), 0, dtype=self.dtype) for num_samples in num_samples_per_bucket]
```

Comment:
If "metadata" always refers to length information, can we rename this to something like `data_prepended_length`? Can we update all instances of "meta" or "metadata" to use the same convention?